libs/datamodel/core/src/common/default_names.rs

```diff
@@ -0,0 +1,20 @@
+pub struct RelationNames {}
+
+impl RelationNames {
+    /// generates a name for relations that have not been explicitly named by a user
+    pub fn name_for_unambiguous_relation(from: &str, to: &str) -> String {
+        if from < to {
+            format!("{}To{}", from, to)
+        } else {
+            format!("{}To{}", to, from)
+        }
+    }
+
+    pub fn name_for_ambiguous_relation(from: &str, to: &str, scalar_field: &str) -> String {
```

Comment:
Did you mean to add a doc comment here too?

Reply:
Wanted to but couldn't think of a useful explanation that says more than the function name so far.